deliravision/torch/models/basic_networks/classification.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from abc import abstractmethod
 import torch
-from delira.models.abstract_network import AbstractPyTorchNetwork
+from delira.models.backends.torch.abstract_network import AbstractPyTorchNetwork
 
 
 class BaseClassificationTorchNetwork(AbstractPyTorchNetwork):
```

Comment:
Do we still need this? Or will we provide another way to wrap pre-implemented classifiaction networks?

Reply:
I also thought about this, but wasn't shure how what the alternative is. I think, I would prefer that the network has a model instead of subclassing the network for every model. This would decouple them from each, but might be a bit less intuitive. What do you think about this?

Reply:
I think I would remove it for now, and reimplement them from scratch witch some rather good default values for both cases. Decoupling them seems nice, but we could also do this via a convert function, which essentially updates the `__dict__` of the `AbstractPyTorchNetwork`. I don't know, which way would be more elegant or straight-forward.